### PR TITLE
Basic command file added

### PR DESCRIPTION
**Pull request by ghost**
a documentation is added regarding helping someone how to search by programming language, organization, updated and created time, by labels (other than good first), by interactions, etc.

### CODE REVIEW

File: .vscode/settings.json
Comment (antonio-pedro99) [inline .vscode/settings.json:1]: I don't think we need this file.
Could you please remove the entire .vscode? It is IDE related configuration.
See why it is a bad idea to have IDE related configuration files: https://bobbyhadz.com/blog/what-is-vscode-folder

Comment (antonio-pedro99) [inline .vscode/settings.json:1]: We do not need this file IMHO.
I should avoid commit IDE/Editor configuration related files.

File: command docs.txt
Comment (antonio-pedro99) [inline command docs.txt:1]: I think this should be a `.md` file and not `.txt`

File: pic1.png
Comment (antonio-pedro99) [inline pic1.png:1]: I do not think you got me. 
We do not need this image.

**Review by antonio-pedro99 — CHANGES_REQUESTED** [review]
Hey @capedCrusaderr, thanks for the PR.

I do not think you got what we need to do.
We do not need any new file. We just need to update the current [README.md](https://github.com/angolaosc/savanna-bot/blob/main/README.md) file with instructions to use the bot.
Why would we use image?
Currently, there are instructions to add the bot to a server only, we do not have instructions showing how to use the bot in the server. This is what we need :)

**Comment by ghost** [comment]
Now i get it , sorry for the trouble . I will work on it

File: pic1.png
Comment (ghost) [reply pic1.png:1]: okay , that's good enough to review and make changes . thank you for the guide.

Comment (antonio-pedro99) [reply pic1.png:1]: I am here for any assistance,  feel free to comment your concerns

**Comment by antonio-pedro99** [comment]
related to #20 

@capedCrusaderr Thanks for your time and patient, this issue has been fixed now.

**Pull request closed without merging**